.travis.yml

```diff
@@ -32,37 +34,52 @@ matrix:
   include:
     # Linux {
     - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=default
+      env: PROJECT_DIR=examples/chaiscript TOOLCHAIN=gcc-5-cxx14-c11
+## Unresolved build error with GCC-7.
```

Comment:
There is no such toolchain in `.travis.yml` template, if this toolchain is not working just remove it.